package.json

```diff
@@ -12,7 +12,12 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint --no-fix"
+    "build-and-test": "NODE_ENV=test npm run build && npm test",
+    "lint": "vue-cli-service lint --no-fix .",
```

Comment:
Why is the lint check changed to include a period at the end?

Reply:
the last argument should be a folder to check, `.` means current folder here